github/github_test.go

```diff
@@ -538,6 +538,42 @@ func TestCheckResponse(t *testing.T) {
 	}
 }
 
+func TestCheckResponse_withBlock(t *testing.T) {
```

Comment:
maybe just add a "block" to the existing `TestCheckResponse`?  All it's really doing is verifying that the JSON is unmarshaled correctly, so splitting these into separate tests doesn't gain much.